docs/reference/commandline/daemon.md

```diff
@@ -537,6 +538,20 @@ please check the [run](run.md) reference.
 daemon instance should use when advertising itself to the cluster. The daemon
 should be reachable by remote hosts on this 'host:port' combination.
 
+The daemon uses [libkv](https://github.com/docker/libkv/) to advertise
+the node within the cluster.  Some Key/Value backends support mutual
+TLS, and the client TLS settings used by the daemon can be configured
+using the `--cluster-store-opt` flag, specifying the paths to PEM encoded
+files. For example:
```

Comment:
The options are fairly straightforward, so the descriptions doesn't have to be long, but for reference; https://github.com/docker/docker/blob/master/docs/reference/commandline/daemon.md#storage-driver-options